src/azure-cli-core/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.34.1
+++++++
+* No changes
+
```

Comment:
Please do not modify history notes manually. It will be automatically generated by CI pipeline when releasing